press/press/doctype/server/server.py

```diff
@@ -799,6 +807,26 @@ def size_to_increase_by_for_20_percent_available(self):  # min 50 GB, max 250 GB
 	def calculated_increase_disk_size(self):
 		self.increase_disk_size(self.size_to_increase_by_for_20_percent_available)
 
+	def prune_docker_system(self):
+		frappe.enqueue_doc(
+			self.doctype,
+			self.name,
+			"_prune_docker_system",
+			queue="long",
+			timeout=4000,
+		)
+
+	def _prune_docker_system(self):
+		self.break_glass()
+		try:
+			ansible = Ansible(
+				playbook="docker_system_prune.yml",
+				server=self,
+			)
+			ansible.run()
+		except Exception:
+			log_error("Prune Docker System Exception", server=self.as_dict())
```

Comment:
Consider using kwarg `doc=self` instead of `server=self.as_dict()`.

Reply:
done